src/WebJobs.Script/Workers/WorkerConstants.cs

```diff
@@ -25,6 +25,7 @@ public static class WorkerConstants
         public const string WorkerDescription = "description";
         public const string ProcessCount = "processOptions";
         public const string WorkerDescriptionArguments = "arguments";
+        public const string WorkerDescriptionWorkerArguments = "workerArguments";
```

Comment:
Would like a discussion on how to name this. From a `worker.config.json` file, the "arguments" property actually gets set in "ExecutableArguments" in `RpcWorkerConfigFactory`, and "workerArguments" is a separate field used later in `DefaultWorkerProcessFactory` as optional args. 

This is just a placeholder for now because WorkerDescriptionArguments and WorkerDescriptionWorkerArguments is definitely confusin...